src/plots/cartesian/axes.js

```diff
@@ -1203,6 +1210,46 @@ axes.calcTicks = function calcTicks(ax, opts) {
     return ticksOut;
 };
 
+function syncTicks(ax) {
+    // get the overlaying axis
+    var baseAxis = ax._mainAxis;
+
+    var ticksOut = [];
+    if(baseAxis && baseAxis._vals) {
```

Comment:
Wondering if we could simplify this to be
```suggestion
    if(baseAxis._vals) {
```
?